tests/components/loqed/test_lock.py

```diff
@@ -0,0 +1,87 @@
+"""Tests the lock platform of the Loqed integration."""
+from loqedAPI import loqed
+
+from homeassistant.components.loqed import LoqedDataCoordinator
+from homeassistant.components.loqed.const import DOMAIN
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_LOCK,
+    SERVICE_OPEN,
+    SERVICE_UNLOCK,
+    STATE_LOCKED,
+    STATE_UNLOCKED,
+)
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_lock_entity(
+    hass: HomeAssistant,
+    integration: MockConfigEntry,
+) -> None:
+    """Test the lock entity."""
+    entity_id = "lock.loqed_lock_home"
+
+    state = hass.states.get(entity_id)
+
+    assert state
+    assert state.state == STATE_UNLOCKED
+
+
+async def test_lock_responds_to_bolt_state_updates(
+    hass: HomeAssistant, integration: MockConfigEntry, lock: loqed.Lock
+) -> None:
+    """Tests the lock responding to updates."""
+    coordinator: LoqedDataCoordinator = hass.data[DOMAIN][integration.entry_id]
+    lock.bolt_state = "night_lock"
+    coordinator.async_update_listeners()
```

Comment:
Move time forward to trigger an update instead.

https://github.com/home-assistant/core/blob/98a94fea99a55764235c24afd02787da898710c3/tests/common.py#L394-L397